integration/tests/cook/test_cli.py

```diff
@@ -900,7 +900,9 @@ def test_show_progress_message(self):
             self.assertEqual(0, cp.returncode, cp.stderr)
             self.assertIn("99% We are so close!", cli.stdout(cp))
 
-            time.sleep(1.0) # allow the executor to wrap up
+            self.logger.info('Sleeping 5 seconds to allow cook executor to complete execution')
+            time.sleep(5.0) # allow the executor to wrap up
+
```

Comment:
Can't we use the `wait_until` approach instead of sleeping?

Reply:
Done.